common/app/conf/switches/IdentitySwitches.scala

```diff
@@ -51,7 +31,7 @@ trait IdentitySwitches {
     "If switched on, users will see UI for opting in or out of different communication channels",
     owners = Seq(Owner.withGithub("walaura")),
     safeState = Off,
-    sellByDate = new LocalDate(2018, 2, 1),
+    sellByDate = new LocalDate(2018, 3, 1),
```

Comment:
I think this one we can either get rid of or make permanent too.  Do you think there's any value in turning this on and off in the future? I'm leaning towards no.